packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -57,6 +59,14 @@ export interface DiscoverySearchProps {
    * overrideComponentSettings is used to override internal collections result state
    */
   overrideComponentSettings?: DiscoveryV2.ComponentSettingsResponse;
+  /**
+   * overrideListFieldsParams is used to override internal listFieldsParams value. See the Discovery Node SDK ListFieldsParams type for more info
+   */
+  overrideListFieldsParams?: Partial<DiscoveryV2.ListFieldsParams>;
+  /**
+   * overrideProjectFields is used to override internal listFields state
+   */
+  overrideListFieldsResponse?: DiscoveryV2.ListFieldsResponse;
```

Comment:
Maybe `overrideListFieldsResults` or `overrideFieldsResults` here for consistency with `overrideCollectionsResults` and 'overrideAutocompletionsResults` above?